packages/datadog-instrumentations/src/mongoose.js

```diff
@@ -19,7 +19,7 @@ function wrapAddQueue (addQueue) {
 
 addHook({
   name: 'mongoose',
-  versions: ['>=4.6.4 <7'] // TODO: Mongoose v7 compat
+  versions: ['>=4.6.4 <5', '5', '6', '>=7']
```

Comment:
Why break out `5` and `6` like this? Couldn't the `<5` just be extended? And why the `>=7`? This could all be reduced to a single `>=4.6.4`, if I understand correctly?

Reply:
This was just a quick and easy way to add tests for those versions. I can add them to `externals.json` if you prefer this to be out of `src`.

Reply:
I don't see any test changes related to that. Did you forget to `git add` something?

Reply:
Not sure I understand, there were definitely multiple test changes that I see in GitHub 🤔